src/providers/fontshare.ts

```diff
@@ -95,6 +95,7 @@ function isFontshareFont (family: string) {
 
 async function getFontDetails (family: string, variants: ResolveFontFacesOptions) {
   // https://api.fontshare.com/v2/css?f[]=alpino@300
+  variants.weights = variants.weights.map(Number)
```

Comment:
I think it would be safer to convert to strings in case something like ‘bold’ is provided.

Reply:
Right. Fixed now.